src/vs/workbench/browser/parts/compositeBar.ts

```diff
@@ -134,6 +142,18 @@ export class CompositeBar extends Widget implements ICompositeBar {
 					}
 				}
 			}
+
+			if (this.compositeTransfer.hasData(DraggedViewIdentifier.prototype)) {
```

Comment:
It feels like we are duplicating this code in the `compositeBar` and `compositeBarActions`. Is it possible to have this code unified in one location?